<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TCMainWindow</class>
 <widget class="QMainWindow" name="TCMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tug Company Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="userType">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1021</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QTableWidget" name="cashierTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>1011</width>
        <height>381</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Ship Name(Zh)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ship Name(En)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Application ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Submission Time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ship Agency</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="welcomeSign0">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>681</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>欢迎</string>
      </property>
     </widget>
     <widget class="QPushButton" name="logOut0">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>0</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>420</y>
        <width>431</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="lastPage0">
         <property name="text">
          <string>Previous Page</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="currentPage0">
         <property name="text">
          <string>Current Page</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nextPage0">
         <property name="text">
          <string>Next Page</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="totalPage0">
         <property name="text">
          <string>Total</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QTableWidget" name="auditorTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>991</width>
        <height>401</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Ship Name(Zh)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ship Name(En)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Application ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ship Agency</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Agent Company</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Billing Clerk</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="welcomeSign1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>欢迎</string>
      </property>
     </widget>
     <widget class="QPushButton" name="logOut1">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>0</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>430</y>
        <width>361</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="lastPage1">
         <property name="text">
          <string>Previous Page</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="currentPage1">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nextPage1">
         <property name="text">
          <string>Next Page</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="totalPage1">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
